pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -501,6 +501,21 @@ func networksAreDisjointed(seed *gardencorev1beta1.Seed, shoot *gardencorev1beta
 		errorMessages = append(errorMessages, e.ErrorBody())
 	}
 
+	if shoot.Status.Networking != nil {
+		for _, e := range cidrvalidation.ValidateMultiNetworkDisjointedness(
+			field.NewPath(""),
+			shoot.Status.Networking.Nodes,
+			shoot.Status.Networking.Pods,
+			shoot.Status.Networking.Services,
```

Comment:
I'm wondering why this check was introduced. The `Shoot.status.networking` section should never be set during scheduling, correct? 😄

Reply:
Yes, indeed, the status should be empty at this point in time.
It was introduced analogous to the validation above without checking the context (shoot scheduling).
Feel free to remove it in a pull request. It should not do any harm as it is, though.